apps/artemis-web/components/hijack-info/hijack-info.tsx

```diff
@@ -20,6 +20,8 @@ class HijackInfoComponent extends Component<any, any> {
   commentRef: any;
   isMobile: any;
   selectRef: any;
+  hijackInfoRight: any;
```

Comment:
what is right and left?

Reply:
The hijack table is split in half to fit in screen.

The left side contains some key-value pairs and the same does the right side